pages/vi/vi-github-and-markdown.md

```diff
@@ -80,7 +80,7 @@ Please, make sure to ALWAYS work on your own personal account via forking and pu
 
 ![New Branch](images/vi-new-branch.png)
 
- To create your page click on the pages/profiles folder. Then, click New File and name the file exactly the same as your GitHub username. We created pages for you using your GitHub username, so it is important that you stick with that name. For reference, an example of a link to a personal page is [kylemathias.github.io/blob/master/pages/profiles/kylemathias.md](https://github.com/kylemathias/kylemathias.github.io/blob/master/pages/profiles/kylemathias.md), so yours should be named in the same manner, only using your username.
+ To create your page click on the pages/vi/profiles folder. Then, click New File and name the file exactly the same as your GitHub username. We created pages for you using your GitHub username, so it is important that you stick with that name. For reference, an example of a link to a personal page is [kylemathias.github.io/blob/master/pages/vi/profiles/kylemathias.md](https://github.com/kylemathias/kylemathias.github.io/blob/master/pages/vi/profiles/kylemathias.md), so yours should be named in the same manner, only using your username.
```

Comment:
This link seems not working anymore because the repo isn't up-to-date with `open-learning-exchange/open-learning-exchange.github.io`. Would be nice to find a recent one in recent closed PRs.